lingam/utils/__init__.py

```diff
@@ -18,9 +23,244 @@
     "get_sink_variables",
     "get_exo_variables",
     "find_all_paths",
+    "simulate_dag",
+    "simulate_parameter",
+    "simulate_linear_sem",
+    "count_accuracy",
+    "set_random_seed",
 ]
 
 
+def set_random_seed(seed):
+    random.seed(seed)
+    np.random.seed(seed)
+
+
+def simulate_linear_sem(W, n, sem_type, noise_scale=1.0):
+    """Simulate samples from linear SEM with specified type of noise.
+
+    Args:
```

Comment:
The docstring of the method added to `__init__.py` is not in Numpy style.

Reply:
I have uploaded the newly-revised __init__.py in https://github.com/YanaZeng/lingam/tree/add-lina-algor/lingam/utils. Please see it.